tailwind.config.js

```diff
@@ -226,6 +226,8 @@ module.exports = {
         sswRed: "#cc4141",
         sswDarkRed: "#8e2c2c",
         sswBlack: "#333333",
+        sswGray: "#797979",
```

Comment:
```suggestion
```

We already have a `ssw-gray` variable identical to this below, better to use that instead

Reply:
right, I just removed it.